Readme.md

```diff
@@ -31,18 +36,18 @@ Note:
 This app outputs:
 1. Summary csv file containing separate performance metrics for SNPs and Indels
 2. Detailed results folder containing:
```

Comment:
does this folder contain more than that stated? raise #18

Reply:
Yes it does. I've updated the README to address this. See https://github.com/moka-guys/dnanexus_happy/pull/16/commits/7a0873ae5f39a93adf18d3b18a8835ecc467c5b6